home_assistant_glow.yaml

```diff
@@ -119,13 +119,20 @@ sensor:
       then:
         - light.turn_on:
             id: led_red
-        - delay: 0.5s
+        - delay: 0.2s
```

Comment:
At least for me, having `0.5s` is too long. With a pulse rate of `1000`, a power consumption of 7200 W means that the LED will be lit up all the time. I have been using `0.2s` since I started using this, and at least for me, it's more than enough for visual inspection of the LED.

Reply:
Totally agree with you to make this value smaller